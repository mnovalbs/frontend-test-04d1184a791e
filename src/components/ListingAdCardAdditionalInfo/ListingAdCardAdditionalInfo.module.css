.additionalInfo {
  font-weight: 400;
  line-height: 22px;
}
