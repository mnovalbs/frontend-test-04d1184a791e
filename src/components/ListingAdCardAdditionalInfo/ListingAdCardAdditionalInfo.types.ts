import { Ad } from "types/Ad";

export interface ListingAdCardAdditionalInfoProps {
  projectType: Ad["project_type"];
  year: Ad["year"];
  ownershipType: Ad["ownership_type"];
  availabilitiesLabel: Ad["availabilities_label"];
}
