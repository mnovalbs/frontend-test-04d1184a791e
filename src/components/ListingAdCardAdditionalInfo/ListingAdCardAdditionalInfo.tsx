import { ListingAdCardAdditionalInfoProps } from "./ListingAdCardAdditionalInfo.types";

function ListingAdCardAdditionalInfo({
  projectType,
  year,
  ownershipType,
  availabilitiesLabel,
}: ListingAdCardAdditionalInfoProps) {
  return (
    <div>
      <div>
        {projectType} · {year} · {ownershipType}
      </div>
      <div>{availabilitiesLabel}</div>
    </div>
  );
}

export default ListingAdCardAdditionalInfo;
