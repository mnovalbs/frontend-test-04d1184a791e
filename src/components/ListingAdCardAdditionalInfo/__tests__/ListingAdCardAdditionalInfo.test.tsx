import { render } from "@testing-library/react";
import mockAd from "contants/mockAd";
import ListingAdCardAdditionalInfo from "../ListingAdCardAdditionalInfo";

describe("ListingAdCardAdditionalInfo", () => {
  it("renders ListingAdCardAdditionalInfo", () => {
    const { container } = render(
      <ListingAdCardAdditionalInfo
        projectType={mockAd.project_type}
        year={mockAd.year}
        ownershipType={mockAd.ownership_type}
        availabilitiesLabel={mockAd.availabilities_label}
      />
    );

    expect(container).toMatchSnapshot()
  });
});
