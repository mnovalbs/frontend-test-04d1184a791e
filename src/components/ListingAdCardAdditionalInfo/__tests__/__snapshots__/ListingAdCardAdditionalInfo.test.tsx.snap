// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListingAdCardAdditionalInfo renders ListingAdCardAdditionalInfo 1`] = `
<div>
  <div>
    <div>
      Condo
       · 
      2021
       · 
      Freehold
    </div>
    <div>
      231 units · 1, 2, 3, 4 bedrooms
    </div>
  </div>
</div>
`;
